packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -36,4 +37,70 @@ export type TPasswordResetAndTradingPasswordManager = {
     email: string;
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
+    toggleModal?: () => void;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: () => void;
+};
+
+export type TCFDPasswordReset = {
+    sendVerifyEmail: () => Promise<VerifyEmailResponse>;
+    account_type: string;
+    account_group: 'real' | 'demo';
+    server: string;
+    password_type: string;
+};
+
+export type TCFDPasswordSuccessMessage = {
+    toggleModal: () => void;
+    is_investor: boolean;
+};
+
+export type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TCFDPasswordManagerTabContentWrapper = {
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    steps: Array<{ component: JSX.Element }>;
+};
+
+export type TInvestorPasswordManager = {
+    error_message_investor: string;
+    is_submit_success_investor: boolean;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    onSubmit: (values: any) => Promise<void>;
+    setPasswordType: (value: string) => void;
+    toggleModal: () => void;
+    validatePassword: (values: { old_password: string; new_password: string; password_type: string }) => void | object;
+};
+
+export type TCFDPasswordManagerTabContent = {
+    toggleModal: () => void;
+    selected_login: string;
+    email: string;
+    setPasswordType: (value: string) => void;
+    multi_step_ref: React.MutableRefObject<any>;
+    platform: 'dxtrade' | 'mt5';
+    onChangeActiveTabIndex: (value: number) => void;
+    account_group: 'real' | 'demo';
+    password_type?: string;
```

Comment:
we do not use password_type in the CFDPasswordManagerTabContent component, can we remove it from props of component and?

Reply:
password_type is used on Line 476 (new code).

Reply:
we pass it as a props to the `CFDPasswordManagerTabContent`, but never use it in the component, so it can be removed.

Reply:
You're right. I removed it from the types and also from the component.